.gitignore

```diff
@@ -1,4 +1,3 @@
-.bundle
```

Comment:
I was about to ask what the `.bundle` directory contains apart from the config. That's a good sounding approach to avoid any mess.